mdx-models/src/main/java/com/mx/path/model/mdx/model/Resources.java

```diff
@@ -249,10 +250,12 @@ public static void registerResources(GsonBuilder builder) {
     builder.registerTypeAdapter(MfaChallenge.class, new ModelWrappableSerializer("mfa_challenge"));
     builder.registerTypeAdapter(new TypeToken<MdxList<MfaChallenge>>() {
     }.getType(), new ModelWrappableSerializer("mfa_challenges"));
-    //ResetPassword
+    // ResetPassword
     builder.registerTypeAdapter(ResetPassword.class, new ModelWrappableSerializer("reset_password"));
-    //ForgotUsername
+    // ForgotUsername
     builder.registerTypeAdapter(ForgotUsername.class, new ModelWrappableSerializer("forgot_username"));
+    // UnlockUser
+    builder.registerTypeAdapter(UnlockUser.class, new ModelWrappableSerializer("unlock"));
```

Comment:
Nitpick: we should probably update the spec and this expected json root to be `unlock_user` as `unlock` is very generic and does not indicate what is being unlocked. We don't call `reset_password` `reset` or `forgot_username` `forgot`.

I strongly suggest updating the spec and updating this root name so it matches the object it actually is

Reply:
Hi @meotch, this is ready for review now that the spec changes have been updated.